core-bundle/tests/Fixtures/Migration/FooRecordedMigration.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Fixtures\Migration;
+
+use Contao\CoreBundle\Migration\AbstractRecordedMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+
+class FooRecordedMigration extends AbstractRecordedMigration
```

Comment:
Thanks to anonymous classes you don't really need this fixture.
You can just use this in your tests:

```php
new class extends AbstractRecordedMigration
{
    public function run(): MigrationResult
    {
        return $this->createResult(true);
    }
}
```

Just as a hint :)

Reply:
Ah you're mocking stuff. That's a different story.

Reply:
With an anonymous class returned a random string for the class name - but I need to know the exact name for the tests.

Reply:
https://github.com/contao/contao/blob/4.x/core-bundle/tests/Routing/ResponseContext/ResponseContextTest.php#L107-L116 ;)

Reply:
Right, ok :). I can still change this if needed. Personally I think it's more convenient this way.